comp/remote-config/rcclient/rcclient.go

```diff
@@ -104,25 +103,58 @@ func (rc rcClient) agentConfigUpdateCallback(updates map[string]state.RawConfig)
 		return
 	}
 
-	// TODO RCM-1064: implement priority between CLI and remote-config
-	// If there is no error, override the configs
-	if len(mergedConfig.LogLevel) > 0 && mergedConfig.LogLevel != rc.configState.LatestLogLevel {
-		pkglog.Infof("Changing log level to %s through remote config", mergedConfig.LogLevel)
+	// Checks who (the source) is responsible for the last logLevel change
+	// The priority between sources is: CLI > RC > Default
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 Praise:** ‏It's nice to document it in the code, thanks